client/my-sites/jetpack-search/placeholder.tsx

```diff
@@ -35,17 +30,14 @@ export default function JetpackSearchPlaceholder( { siteId }: Props ): ReactElem
 			<DocumentHead title="Jetpack Search" />
 			<SidebarNavigation />
 
-			<FormattedHeader
-				headerText={ translate( 'Jetpack Search' ) }
-				id="jetpack-search-header"
-				align="left"
-				brandFont
+			<JetpackSearchContent
+				headerText={ translate( 'Finely-tuned search for your site.' ) }
+				bodyText={ translate( 'Your visitors are getting our fastest search experience.' ) }
+				buttonLink={ null }
```

Comment:
`buttonLink` is typed `TranslateResult` or `undefined`, a `null` here shows a warning on my editor. And the same case is with `onClick` too.

There are some cases other places as well, not sure if this is something we should worry about.